requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Use >= for development versions so that source builds always work
-coala>=0.8.0
+coala>=0.9.0.dev20160902132949
```

Comment:
please upgrade this to the latest which will fix the bug that you hit and it will correctly ask you about missing settings - also if you could try it out that'd be nice as well :)